Assets/Scripts/ListComponents/TasksView/TasksUIController.cs

```diff
@@ -1,3 +1,29 @@
+using UnityEngine;
+using UnityEngine.Scripting;
+
 public class TasksUIController : ListUIController<Task>
 {
+    private GameObject newTaskButton;
+    private GameObject taskList;
+
+    protected override void Awake()
+    {
+        base.Awake();
+        newTaskButton = GameObject.Find("NewTaskButton");
```

Comment:
Instead of .Find() it is better to set the variable to public or add [SerializeField] and simply assign the reference to NewTaskButton in the inspector.